plugins/hls-refactor-plugin/src/Development/IDE/GHC/ExactPrint.hs

```diff
@@ -374,7 +382,7 @@ graftWithM dst trans = Graft $ \dflags a -> do
 #if MIN_VERSION_ghc(9,2,0)
                                 val'' <-
                                     hoistTransform (either Fail.fail pure) $
-                                        annotate dflags True $ maybeParensAST val'
+                                        annotate dflags False $ maybeParensAST val'
```

Comment:
I made this change because there was a failing test which was expanding a type signature to:

```
f ::  Int
```

instead of,

```
f :: Int
```

(note the extra space before `Int.)

I'm not sure what changed to make this differ from the pre-9.2 change, and it might not be the right fix to be honest, but since I think it just messes up formatting, and I find it very difficult to debug this exactprint annotation stuff, I think it's either this or nothing, unless someone else wants to dig in!